src/WikiService.Validation/Rubric/EditRubricRequestValidator.cs

```diff
@@ -0,0 +1,195 @@
+﻿using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.WikiService.Data.Interfaces;
+using LT.DigitalOffice.WikiService.Models.Db;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests.Rubric;
+using LT.DigitalOffice.WikiService.Validation.Rubric.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.WikiService.Validation.Rubric
+{
+  public class EditRubricRequestValidator : ExtendedEditRequestValidator<Guid, EditRubricRequest>, IEditRubricRequestValidator
+  {
+    private readonly IRubricRepository _rubricRepository;
+
+    private async Task HandleInternalPropertyValidationAsync(
+      Operation<EditRubricRequest> requestedOperation,
+      CustomContext context)
+    {
+      Context = context;
+      RequestedOperation = requestedOperation;
+
+      #region paths
+
+      AddСorrectPaths(
+        new List<string>
+        {
+          nameof(EditRubricRequest.Name),
+          nameof(EditRubricRequest.ParentId),
+          nameof(EditRubricRequest.IsActive)
+        });
+
+      AddСorrectOperations(nameof(EditRubricRequest.Name), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.ParentId), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditRubricRequest.IsActive), new List<OperationType> { OperationType.Replace });
```

Comment:
it's possible to change "new List<OperationType>" to "new()" - result would be the same
"new()" is shorter;  "new List<OperationType>" is more understandable;
i'm just suggesting$ it's your choise =)